db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```

Comment:
Nice :), as a suggestion you can also make use of a cycle to create those objects, just interpolate the index on the name or email or any field you want to make them different 🤓 👍

Reply:
Ok, i will take note about this suggestions and will add in the next commit